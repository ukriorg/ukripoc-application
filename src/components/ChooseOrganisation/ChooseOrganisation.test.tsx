import React from "react";
import renderer from "react-test-renderer";
import { ChooseOrganisation } from "./index";

describe("ChooseOrganisation", () => {
    it("renders correctly", () => {
        const tree = renderer.create(<ChooseOrganisation />).toJSON();
        expect(tree).toMatchSnapshot();
    })
});

