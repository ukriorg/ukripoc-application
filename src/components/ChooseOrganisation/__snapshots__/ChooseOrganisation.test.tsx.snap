// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChooseOrganisation renders correctly 1`] = `
<p>
  Org 
  <button
    onClick={[Function]}
  />
</p>
`;
